src/app/lib/agoraAPI.js

```diff
@@ -5,7 +5,7 @@ import { NextResponse } from "next/server";
  */
 class AgoraAPI {
   /**
-   * Initializes a new instance of the AgoraAPI class.
+   * Initializes a create instance of the AgoraAPI class.
```

Comment:
can you explain why there are many instances of changing _"new"_ to _"create"_ here? In my head it still sounds correct to initialize a new instance (you typically do `new Instance()` but the frequency at which I'm seeing this comment makes me think you deliberately changed something that I might be missing